pype_schema/parse_json.py

```diff
@@ -572,10 +657,16 @@ def create_connection(self, connection_id, node_obj):
                                 tag_obj.contents.name,
                                 tag.tag_type.name,
                             )
-                            v_tag = VirtualTag(tag_id, tag_list, binary_operations="+")
+                            v_tag = VirtualTag(
+                                tag_id,
+                                tag_list,
+                                operations=operations,
+                                units=tag_obj.units,
+                            )
                             connection_obj.add_tag(v_tag)
                     if "total" not in tag_dest_unit_ids and len(tag_dest_unit_ids) > 1:
                         tag_obj = tags_by_contents[0]
+                        operations = utils.get_tag_sum_lambda_func(tag_dest_unit_ids)
```

Comment:
here is an example where if two connections have `dest_unit_id="total"` then you get an error in the lambda function. E.g., the total outflow from the cogenerator is the sum of `Cogenerator_VirtualDemand_Electricity_Flow` and `SVCW_Cogenerator_PowerGrid_Electricity_Flow` which are both `"total"` destination unit IDs